cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -15,7 +15,12 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
-// CNV-2928/CNV-4166 OVN-Kubernetes network provider
+[id="cnv-2-3-networking"]
+=== Networking
+
+* The Open Virtual Networking (OVN) Kubernetes network plug-in is a network virtualization platform developed by the Open vSwitch community.
```

Comment:
I'm not sure we have to define ovn here. I think that the xref is enough.

But even with this, the PR is acceptable.